instrumentation/apache-httpclient/apache-httpclient-5.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/apachehttpclient/v5_0/ApacheHttpAsyncClientInstrumentation.java

```diff
@@ -159,7 +160,7 @@ public void sendRequest(HttpRequest request, EntityDetails entityDetails, HttpCo
     private static final Logger logger = Logger.getLogger(WrappedFutureCallback.class.getName());
 
     private final Context parentContext;
-    private final HttpContext httpContext;
+    @Nullable private final HttpContext httpContext;
```

Comment:
nullable on the field in the other class too?

Reply:
Doh! yeah...will do.